packages/webapp/src/views/land/check-and-submit.html

```diff
@@ -50,6 +50,23 @@ <h2 class="govuk-heading-l">Your details</h2>
                   }
                 ]
               }
+            },
+            {
+              key: {
+                text: "Email address"
+              },
+              value: {
+                text: application.applicant.email
+              },
+              actions: {
+                items: [
+                  {
+                    href: routes.CORRECT_OWNER_EMAIL,
```

Comment:
According to Dave Hunter's comment when editing the email address it should take you to the /email route not the /correct-email route https://eaflood.atlassian.net/browse/BNGP-1099?focusedCommentId=453117

Reply:
The implementation is in alignment with Dave's comment